celeri/get_mesh_edge_elements.py

```diff
@@ -0,0 +1,56 @@
+def get_mesh_edge_elements(meshes):
+    # Find indices of elements lining top, bottom, and sides of each mesh
+    
+	from ismember import ismember
+	
+	celeri.get_ordered_edge_nodes(meshes)
+	
+	for i in range(meshes):
+		coords = meshes[i].meshio_object.points
+		vertices = meshes[i].verts
+		# Arrays of all element side node pairs
+		side_1 = np.sort(np.vstack((vertices[:, 0], vertices[:, 1])).T, 1)
+		side_2 = np.sort(np.vstack((vertices[:, 1], vertices[:, 2])).T, 1)
+		side_3 = np.sort(np.vstack((vertices[:, 2], vertices[:, 0])).T, 1)
+		# Sort edge node array
+		sorted_edge_nodes = np.sort(meshes[i].ordered_edge_nodes, 1)
+		# Indices of element sides that are in edge node array
+		side_1_in_edge, side_1_in_edge_idx = ismember(sorted_edge_nodes, side_1, 'rows')
```

Comment:
We're using double quotes throughout for all strings.  One way to enforce this is to install the "black" autoformatter (https://formulae.brew.sh/formula/black) locally and run this before submitting.  I followed these instructions (https://dev.to/adamlombard/how-to-use-the-black-python-code-formatter-in-vscode-3lo0) to have VSCode automatically format all .py files using the black autoformatter any time I save them.  This does not apply to ipynb files.

Reply:
Edited (saved with black formatting) in a876f1233bb3b0e1b63337fd2e8b9800816d7e6c